packages/date-picker/src/vaadin-date-picker-overlay-content-mixin.js

```diff
@@ -736,14 +736,6 @@ export const DatePickerOverlayContentMixin = (superClass) =>
       return this._translateX < this._yearScrollerWidth / 2;
     }
 
-    /** @private */
-    _translateXChanged(x) {
```

Comment:
There is a warning about the observer being not defined, please remove this from the property:

```
 observer: '_translateXChanged',
```